Essentials/src/messages.properties

```diff
@@ -429,21 +413,20 @@ pTimeSetFixed=\u00a76Player time is fixed to \u00a7c{0}\u00a76 for\: \u00a7c{1}.
 pWeatherCurrent=\u00a7c{0}\u00a76''s weather is\u00a7c {1}\u00a76.
 pWeatherInvalidAlias=\u00a74Invalid weather type
 pWeatherNormal=\u00a7c{0}\u00a76''s weather is normal and matches the server.
-pWeatherOthersPermission=\u00a74You are not authorized to set other players'' weather.
+pWeatherOthersPermission=\u00a74You are not authorized to set other players weather.
 pWeatherPlayers=\u00a76These players have their own weather\:\u00a7r
 pWeatherReset=\u00a76Player weather has been reset for\: \u00a7c{0}
 pWeatherSet=\u00a76Player weather is set to \u00a7c{0}\u00a76 for\: \u00a7c{1}.
-questionFormat=\u00a72[Question]\u00a7r {0}
-radiusTooBig=\u00a74Radius is too big\! Maximum radius is {0}.
+radiusTooBig=\u00a74Radius is too big\! Maximum radius is\u00a7c {0}\u00a74.
 readNextPage=\u00a76Type\u00a7c /{0} {1} \u00a76to read the next page.
-realName=\u00a7f{0}\u00a7r\u00a76 is \u00a7f{1}
+realName=\u00a7f{0}\u00a76 is \u00a7f{1}
```

Comment:
I don't think we can quite do this because the display name of a user may contain underline or magic in them.